core/templates/pages/preferences-page/preferences-page.component.ts

```diff
@@ -27,43 +27,50 @@ import { WindowRef } from 'services/contextual/window-ref.service';
 import { I18nLanguageCodeService } from 'services/i18n-language-code.service';
 import { LoaderService } from 'services/loader.service';
 import { PreventPageUnloadEventService } from 'services/prevent-page-unload-event.service';
-import { SubscriptionSummary, UserBackendApiService } from 'services/user-backend-api.service';
+import { EmailPreferencesBackendDict, SubscriptionSummary, UserBackendApiService } from 'services/user-backend-api.service';
 import { UserService } from 'services/user.service';
 import { EditProfilePictureModalComponent } from './modal-templates/edit-profile-picture-modal.component';
 require('cropperjs/dist/cropper.min.css');
 
+type AudioLangaugeChoices = {
+  id: string,
+  text: string
+}[];
+
```

Comment:
I think an interface would be more appropriate here ?

Reply:
done